octobot/exchange_factory.py

```diff
@@ -49,12 +47,13 @@ def __init__(self, octobot, ignore_config=False):
 
         self.available_exchanges = ccxt.exchanges
 
-    async def create(self):
+    async def create(self, tentacles_setup_config):
         if self.octobot.config[CONFIG_EXCHANGES]:
             for exchange_class_string in self.octobot.config[CONFIG_EXCHANGES]:
                 if exchange_class_string in self.available_exchanges:
                     exchange_builder = create_exchange_builder(self.octobot.config, exchange_class_string) \
                                           .has_matrix(self.octobot.evaluator_factory.matrix_id) \
+                                          .use_tentacles_setup_config(tentacles_setup_config) \
```

Comment:
As the above lines we can have : `self.octobot.tentacles_setup_config` instead of `tentacles_setup_config` param